test/algorithms/buffer/buffer_geo_spheroid.cpp

```diff
@@ -57,24 +57,29 @@ void test_point(std::string const& label, Spheroid const& spheroid, double expec
     test_one_geo<Point, polygon>(label, torg, strategy, side, circle, join_round, end_round, expected_area, 100.0, settings);
 }
 
-int test_main(int, char* [])
+template <typename test_type>
+void test_all()
 {
-    BoostGeometryWriteTestConfiguration();
-
-    using test_type = default_test_type;
-
     using point_t = bg::model::point<test_type, 2, bg::cs::geographic<bg::degree>>;
 
     // Use the default spheroid
     bg::srs::spheroid<test_type> def_spheroid;
-    test_linestring<bg::strategy::andoyer, true, point_t>("line_def", def_spheroid, 7996.0, 8093.0);
-    test_point<bg::strategy::andoyer, true, point_t>("point_def", def_spheroid, 31450.0);
+    test_linestring<bg::strategy::andoyer, true, point_t>("line_def", def_spheroid, 8046.0, 8143.0);
+    test_point<bg::strategy::andoyer, true, point_t>("point_def", def_spheroid, 31414.0);
```

Comment:
I don't see what really changed, apart from the expectations.

But you mentioned it's a personal test, so that's fine.